deployment/ansible.cfg

```diff
@@ -1,4 +1,4 @@
 [defaults]
 roles_path = roles/
-inventory = inventory.yaml
+inventory = inventory
```

Comment:
Why no extension?

Reply:
The non-yaml format is simpler for the inventory file. Easier to read!